Automattic/jetpack
# AI Proofread: Improve Popover UX

Improve the Popover UX inside Jetpack AI Proofread

## Proposed changes:

* Handle nested popover with `mouseover` and `timeout` than using the store.
* Implemented Popover to follow cursor if it's a long sentence, more than 4 words (can be improved?)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Open Editor.
* Make sure Breve is enabled.
* Write a post that contains multiple suggestions.
* When hovering the suggestions it should stay the same for one word highlights, or follow the cursor for longe sentences.
